### PR TITLE
Improve automated tests

### PR DESCRIPTION
**Pull request by hanneskaeufler**
#### Reference to a related issue in the repository

None - These are out of context quality of life improvements in the automated test suite

#### Add a description

I saw some long-hanging fruit in improving these tests. I hope the commit messages do
well enough to explain the change/intention behind it. If not, feel free to comment on details :)

#### Check the checklist

- [x] My code and comments follow the [style guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/commenting.html) and [contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/osi/howtocontribute.html) of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the [documentation](https://github.com/OpenSimulationInterface/osi-documentation).
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests / travis ci pass locally with my changes.

### CODE REVIEW

**Comment by jdsika** [comment]
Thank you Hannes for your contribution. It is very welcome! @vkresch will review it tomorrow

File: tests/test_newline.py [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-                for line in fin:
-                    hasNewLine = line.endswith("\n")
-
-                self.assertTrue(hasNewLine, file + " has no new line at the end of the file.")
-                
\ No newline at end of file
+        for file in PROTO_FILES:
+            with open(file, "rt") as fin, self.subTest(file=file):
+                lastCharacter = fin.read()[-1]
+                self.assertEqual(lastCharacter, "\n")
```
Comment (vkresch) [inline tests/test_newline.py]: Please add `self.assertEqual(lastCharacter, "\n", file + " has no new line at the end of the file")` to excplictly warn the user that he is missing a newline character.

Comment (hanneskaeufler) [reply tests/test_newline.py]: Done! Thanks for the review!

**Comment by vkresch** [comment]
https://travis-ci.community/t/known-issue-travis-ci-reports-expected-waiting-for-status-to-be-reported-on-the-github-status-api-but-the-status-never-arrives/1154

**Comment by jdsika** [comment]
I do not find the option. Very strange...

**Comment by jdsika** [comment]
@hanneskaeufler I invited you to the organization. Maybe this is the problem of the status not being properly reported. Can you accept the invite and make an empty commit (not reopen of PR) as a member? If this does not work @vkresch will copy your stuff and make a branch.

**Comment by hanneskaeufler** [comment]
> @hanneskaeufler I invited you to the organization. Maybe this is the problem of the status not being properly reported. Can you accept the invite and make an empty commit (not reopen of PR) as a member? If this does not work @vkresch will copy your stuff and make a branch.

Thanks! I accepted and rebased one more time ...

**Pull request closed without merging**